### DISCUSSION TITLE
Remove Roboto font due to possible GPDR issues?

### DISCUSSION THREAD

**Discussion by Marcuse7** [storybookjs/storybook#15980] [General] [answered]
The storybook examples use Google fonts such as Roboto. If you create a customer or public Storybook site with Google-hosted fonts, you might risk a _cease and desist_ order for GPDR violation, as Google collects your user's data. At least in Germany, the lawyers will also charge you. Would it possible to use different fonts for the examples?

**Comment by MichaelArestad** [comment, marked as answer]
We can avoid any data collection by downloading the fonts from Google and including them in the example Storybooks. I would be happy to point folks in the right direction if they are interested in making this change.

**Comment by ghengeveld** [comment]
We certainly do not want Storybook to get its fonts from Google. If not for the privacy implications, then for the performance implications. Huge no-go.
